sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collationExpressions.scala

```diff
@@ -38,8 +38,8 @@ import org.apache.spark.sql.types._
     Examples:
       > SET spark.sql.collation.enabled=true;
       spark.sql.collation.enabled	true
-      > SELECT COLLATION('Spark SQL' _FUNC_ UTF8_LCASE);
```

Comment:
This example is fine. We need a way to show the specified string collation.

Reply:
And `str COLLATE UTF8_LCASE` is the more standard way to specify collation.

Reply:
In fact, this example is not related to `CollateExpressionBuilder`.